sql/chunk.sql

```diff
@@ -100,3 +100,12 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.attach_osm_table_chunk(
    hypertable REGCLASS,
    chunk REGCLASS)
 RETURNS BOOL AS '@MODULE_PATHNAME@', 'ts_chunk_attach_osm_table_chunk' LANGUAGE C VOLATILE;
+
+-- Trigger that blocks modifications on frozen chunks
+CREATE OR REPLACE FUNCTION _timescaledb_internal.frozen_chunk_modify_blocker() RETURNS trigger
+   LANGUAGE plpgsql STRICT AS
+$BODY$
+BEGIN
+     RAISE EXCEPTION 'Unable to modify frozen chunk %s', TG_TABLE_NAME;
```

Comment:
```suggestion
     RAISE EXCEPTION 'unable to modify frozen chunk %s', TG_TABLE_NAME;
```

Reply:
@erimatnor Thanks for the suggestion, changed it.